app/components/pay-page/choose-membership-plan-modal.ts

```diff
@@ -0,0 +1,134 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import fade from 'ember-animated/transitions/fade';
+import { inject as service } from '@ember/service';
+import Store from '@ember-data/store';
+import window from 'ember-window-mock';
+import type RegionalDiscountModel from 'codecrafters-frontend/models/regional-discount';
+import type PromotionalDiscountModel from 'codecrafters-frontend/models/promotional-discount';
+import type AuthenticatorService from 'codecrafters-frontend/services/authenticator';
+
+interface Signature {
+  Element: HTMLAnchorElement;
+
+  Args: {
+    onClose: () => void;
+    regionalDiscount?: RegionalDiscountModel | null;
+    shouldApplyRegionalDiscount?: boolean;
```

Comment:
Let's remove this and always apply a regional discount - this design isn't intended to be shipped with "optional" regional discounts, it's very easy to miss the initial banner and users will complain later on